inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/trivial_concat.cpp

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
I wrote comment in the previous PR about this new file. These tests should be in `concat.hpp/cpp` files. Please, don't duplicate SLT file. It leads to confusion

Reply:
well this test is completly different than original one, it needs some GNA specific operations and concat.cpp cannot be used to test these cases without fully rewrting the test. Do you suggest to have two separate tests in concat.cpp?

Reply:
the answer for both comments: if it is GNA specific test, it will be better to move it from shared lib. We do not prohibit develop device specific test cases out of `shared_lib` (The examples is `bf16 tests` in cpu and `remote_blob tests` in functional tests). I mean that shared lib is the place where we develop tests which can be executed on more then 2 devices. I see your points about the GNA specific, but IMO all single layer tests should be able to execute on GNA too (internal plugin transformations to change 1 op graph for example). As a result, it's not OK for me to have these tests here.